index.html

```diff
@@ -4007,6 +4007,325 @@ <h2>
+  }
+}
+      </pre>
+
+      <pre class="example" title="Verifiable Credential as Decoded JWT">
+        {
+          "protected": {
+            "kid": "did:example:123#_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A",
+            "alg": "EdDSA"
+          },
+          "payload": {
+            "iss": "did:example:123",
+            "sub": "did:example:456",
+            "vc": {
+              "@context": [
+                "https://www.w3.org/2018/credentials/v1",
+                "https://www.w3.org/2018/credentials/examples/v1"
+              ],
+              "id": "http://example.gov/credentials/3732",
+              "type": [
+                "VerifiableCredential",
+                "UniversityDegreeCredential"
+              ],
+              "issuer": {
+                "id": "did:example:123"
+              },
+              "issuanceDate": "2020-03-10T04:24:12.164Z",
+              "credentialSubject": {
+                "id": "did:example:456",
+                "degree": {
+                  "type": "BachelorDegree",
+                  "name": "Bachelor of Science and Arts"
+                }
+              }
+            },
+            "jti": "http://example.gov/credentials/3732",
+            "nbf": 1583814252
+          },
+          "signature": "qSv6dpZJGFybtcifLwGf4ujzlEu-fam_M7HPxinCbVhz9iIJCg70UMeQbPa1ex6BmQ2tnSS7F11FHnMB2bJRAw"
+        }
+      </pre>
+    </section>
+
+    <section class="informative">
+      <h2>
+        Encrypting
+      </h2>
+      <p class="note">
+        These examples are for information purposes only, 
+        it is considered a best practice to avoid dislosing unnecessary information in JWE headers.
+      </p>
+      <pre class="example" title="JWE linked to a verification method via kid">
+        {
+          "ciphertext": "3SHQQJajNH6q0fyAHmw...",
+          "iv": "QldSPLVnFf2-VXcNLza6mbylYwphW57Q",
+          "protected": "eyJlbmMiOiJYQzIwUCJ9",
+          "recipients": [
+            {
+              "encrypted_key": "BMJ19zK12YHftJ4sr6Pz1rX1HtYni_L9DZvO1cEZfRWDN2vXeOYlwA",
+              "header": {
+                "alg": "ECDH-ES+A256KW",
+                "apu": "Tx9qG69ZfodhRos-8qfhTPc6ZFnNUcgNDVdHqX1UR3s",
+                "apv": "ZGlkOmVsZW06cm9wc3RlbjpFa...",
+                "epk": {
+                  "crv": "X25519",
+                  "kty": "OKP",
+                  "x": "Tx9qG69ZfodhRos-8qfhTPc6ZFnNUcgNDVdHqX1UR3s"
+                },
+                "kid": "did:example:123#zC1Rnuvw9rVa6E5TKF4uQVRuQuaCpVgB81Um2u17Fu7UK"
+              }
+            }
+          ],
+          "tag": "xbfwwDkzOAJfSVem0jr1bA"
+        }
+      </pre>
+    </section>
+
+    <section class="informative">
+      <h2>
+        Zero Knowledge Proofs
+      </h2>
+      <p class="issue">
+        Add BBS+ / HyperLedger Indy examples here, or remove section.
+      </p>
+    </section>
```

Comment:
```suggestion
```

No need to put in a place holder. Just do a separate PR once we have an example.